tests/data_ingestion/test_timeseries_resampler.py

```diff
@@ -0,0 +1,115 @@
+"""
+Tests for the `TimeSeriesResampler`
+
+Copyright
+Copyright © 2021 Frequenz Energy-as-a-Service GmbH
+
+License
+MIT
+"""
+from datetime import datetime, timedelta
+from typing import Sequence
+
+import pytz
+import time_machine
+
+from frequenz.sdk.data_ingestion.resampling.component_metric_resampler import (
+    ComponentMetricResampler,
+    Sample,
```

Comment:
This should be imported from `data_pipeline`, I guess it was missed in some rebasing (this comment is on a particular commit, the commit "Implement a resampler for resampling samples for a single time series" (which I would cut to "Implement a resampler for a single time series" to cut a bit on the resampling samples madness :laughing: ). It was changed in the next commit, if you want to clean the commits great, otherwise we can leave it as it is.

Reply:
Change the commit message and cleaned up the commits (thanks for catching that, I must have messed it up while adding fixups).

Reply:
Yeah, it is very hard not ending up with one of these after so many rebases :)